internal/certgen/certgen.go

```diff
@@ -196,7 +288,10 @@ func OutputCerts(certgenConfig *Config, certs *ContourCerts) error {
 		}
 
 		fmt.Printf("Outputting certs to PEM files in %s/\n", certgenConfig.OutputDir)
-		certs.writeCertPEMs(certgenConfig.OutputDir)
+		err = certs.writeCertPEMs(certgenConfig.OutputDir)
+		if err != nil {
+			return err
+		}
 	}
 
 	if certgenConfig.OutputYAML {
```

Comment:
This feels like selection; it probably shouldn't be a set of if cases where only one matches.

Reply:
I intended that you could pass from one to three, and get all of the results you asked for.